docs/virtual_machines/vms_and_vmis.md

```diff
@@ -0,0 +1,41 @@
+# Virtual Machine types
+There are two main virtual machine user facing API objects: a `VirtualMachine`,
+and a `VirtualMachineInstance`.
+
+## Virtual Machine Instance
+A
+[VirtualMachineInstance](https://kubevirt.io/user-guide/virtual_machines/virtual_machine_instances/)
+represents a single **running** virtual machine.
+
+It is encapsulated in a pod.
+
+When used without a controlling VMI object, it should be limited to running
+stateless workloads.
+
+Removing the VMI object represents stopping the Virtual Machine.
```

Comment:
```suggestion
Removing the VMI object represents stopping the virtual machine.
```
Shouldnt we use VM in context of 'VirtualMachine' (Kubernetes object) and virtual machine as something that is started in a Pod?